.env.template

```diff
@@ -1,2 +1,4 @@
 API=https://host.docker.internal:9000
 FRONTEND_PORT=3000
+NUXT_PUBLIC_SENTRY_DSN=
+NUXT_PUBLIC_SENTRY_ENVIRONMENT=development
```

Comment:
Why we need to set explicitly `development` ? Not better to set nothging or `prod` ?

Reply:
We could set nothing as we already have a default case in `nuxt.config.ts` at `runtimeConfig.public.sentry.environment`.
But this is the `.env.template` so I think it's fine to propose all the keys supported by the app no ?

Reply:
The `.env.template` should work as is when copied as `.env`

Reply:
Isn't it the case ?
When in dev (it's superfluous but works) and in prod we just have to change the value of `NUXT_PUBLIC_SENTRY_ENVIRONMENT` to `production`.

But we could set it to `production` directly if you prefer ?